Decent.Minecraft.Client/WaterLevel.cs

```diff
@@ -3,7 +3,7 @@
     /// <summary>
     /// Species of wood.
     /// </summary>
-    public enum WaterLevel : byte
+    public enum Level : byte
     {
         Source = 0,
```

Comment:
Yeah, I'll do that tonight. I spent time last night trying to dogfood the scripts we have to set up a CI.